images/skopeo-ghcr-io.yaml

```diff
@@ -1,6 +1,7 @@
 # Docs: https://github.com/kubasobon/skopeo/blob/main/docs/skopeo-sync.1.md#yaml-file-content-used-source-for---src-yaml
 ghcr.io:
   images:
+    ahrtr/etcd-defrag: []
```

Comment:
Is this valid? Why not use the `images-by-semver`?

Reply:
https://github.com/kubasobon/skopeo/blob/main/docs/skopeo-sync.1.md#yaml-file-content-used-source-for---src-yaml

> Repository `registry.example.com/busybox`: all images, as no tags are specified.

Reply:
So, yes, I assume this is valid. Or would you really just retag images matching a semver?

Reply:
I mean, do we need the older images? I assume you're currently wanting to use v0.22.0? Lets just retag that and newer?

Reply:
Yeah, makes sense. Also my current approach would probably also retag untagged images or at least development ones.

Reply:
Fixed!